crowdfunding_compassion/static/src/js/creation_form.js

```diff
@@ -62,3 +62,8 @@ odoo.define('crowdfunding_compassion.creation_form', function (require) {
 
     return animation.registry.crowdfunding_creation_form;
 });
+
+$( document ).ready(function() {
+    presentation_input = document.getElementById("presentation_video")
+    presentation_input.placeholder = "e.g. https://vimeo.com/jlkj34ék5"
+});
```

Comment:
This can be put in the attribute `help` of the field `presentation_video` in the model `crowdfunding.participant` and `crowdfunding.project`
```suggestion
```